Libraries/TestLogs.psm1

```diff
@@ -112,7 +112,7 @@ function Collect-TestLogs {
 	$currentTestResult = Create-TestResultObject
 
 	if ($TestType -eq "sh") {
-		$filesTocopy = "./state.txt, ./summary.log, ./TestExecution.log, ./TestExecutionError.log"
+		$filesTocopy = "./state.txt, ./*summary.log, ./TestExecution.log, ./TestExecutionError.log"
```

Comment:
probably it will break some cases.

Reply:
Do you know which test cases may fail?  I can run them locally for testing.